Model/Quote/QuoteAction/Currency.php

```diff
@@ -11,6 +11,7 @@
  */
 class Currency implements QuoteActionInterface
 {
+    private const SET_DISPLAY_CURRENCY = 'SET_DISPLAY_CURRENCY';
```

Comment:
lowercase to match the rest of the events

Reply:
Changed to lowercase